### PR TITLE
update data access docs

### LINKED ISSUE #334: Update documentation to go to ISIS compute
Issue raised by: [developer]

### What?
As a temporary solution after the samba share for data access has been decommissioned we will use ISIS compute and IDAaaS (until the IDAaaS web plugin for file browsing becomes available)
* Remove Samba share related documentation
* Add documentation for data access via SFTP with ISIS Compute
* Add documentation for data access via SFTP with IDAaaS

### Where?
* Documentation on the webapp help pages
* Wiki pages

### How?
Solution to use isis compute was made by discussions with CEPH and IDAaaS representatives

### How to test the issue is resolved
* Ensure that no documentation referring to the samba share still exists.
* Ensure the new documentation for ISIS Compute and IDAaaS data access makes sense

### PR DESCRIPTION
**Pull request by ElliotAOram**
### Summary of work
* Updated documentation pages to use ISIS compute / IDAaaS

### How to test your work
Ensure documentation makes sense

Fixes #334

### CODE REVIEW

File: WebApp/autoreduce_webapp/templates/help.html [hunk 1]
```diff
@@ -19,14 +19,41 @@ <h2>Help Pages</h2>
...
+            <h4>How do I access my reduced data?</h4>
         </div>
         <div class="panel-body">
             <p>
-                The link to reduced data points to a network drive that is accessible from within the RAL site (including by VPN).
-                On Windows, copy the link into the navigation bar at the top of a File Explorer window and hit enter.
-                Alternatively, the network drive (<code>\\data.analysis.stfc.ac.uk\rb\</code>) can be mapped to a location on your disk.
+                In order to copy reduced data from CEPH to your local machine you will need to download an SFTP client.
+                We recommend <a href="https://winscp.net/eng/download.php">WinSCP</a> if you are using a windows operating system.
+                The step below are based on WinSCP as the <code>SFTP Client</code> and assume you have downloaded and installed it.
```
Comment (gc-code) [inline WebApp/autoreduce_webapp/templates/help.html]: 'steps'

File: WebApp/autoreduce_webapp/templates/help.html [hunk 2]
```diff
@@ -19,14 +19,41 @@ <h2>Help Pages</h2>
...
             </p>
+            <h2>IDAaaS</h2>
+            <ol>
+                <li>Visit the IDAaaS page <a href="https://isis.analysis.stfc.ac.uk">https://isis.analysis.stfc.ac.uk</a></li>
+                <li>Create a new IDAaaS instance for your instrument / technique.</li>
+                <li>Follow the instructions on the <a href="https://isis.analysis.stfc.ac.uk/#/help">IDAaaS help page</a>
+                    under the <em>Download/Upload Files</em> section that will walk you through the SFTP process </li>
+            </ol>
+            <h2>ISIS Compute</h2>
+            If you do not currently have your instrument supported by IDAaaS then you should use the ISIS Compute service.
```
Comment (gc-code) [inline WebApp/autoreduce_webapp/templates/help.html]: Is this option accessible from outside the site firewall?

File: WebApp/autoreduce_webapp/templates/help.html [hunk 3]
```diff
@@ -19,14 +19,41 @@ <h2>Help Pages</h2>
...
+                <ul>
+                    <li>To avoid repeating these steps in the future, click save and fill in the save dialog box that opens</li>
+                </ul>
+                <li>Click Login</li>
+                <ul>
+                    <li>You may get a warning message about a potential security breach.
+                        This is nothing to worry about and is common especially if it's your first time accessing the server in a while.
+                        Simply click <strong>Update</strong> to continue.</li>
+                </ul>
+                <li>Once the client has connected to the server, you will see your files (default on the right) and the server files (default on the left)</li>
```
Comment (gc-code) [inline WebApp/autoreduce_webapp/templates/help.html]: My files appeared on the left and the server files appeared on the right when I followed these steps.

File: WebApp/autoreduce_webapp/templates/help.html (hunk 2 above)
Comment (ElliotAOram) [reply WebApp/autoreduce_webapp/templates/help.html]: I don't believe so. I have now added this to the top section stating that you must be onsite/VPN to access

File: WebApp/autoreduce_webapp/reduction_viewer/views.py [hunk 4]
```diff
@@ -367,23 +368,6 @@ def run_summary(request, instrument_name=None, run_number=None, run_version=0):
...
-    :return: string to reduced data storage location in CEPH
-    """
-    ceph_root_level = '//data.analysis.stfc.ac.uk/'
-    try:
-        data_location = reduction_location.split(instrument.upper())[1]
-    except IndexError:
-        # Return None if file path does not contain instrument name
-        return None
-    ceph_location = ceph_root_level + 'rb/' + instrument.upper() + data_location
-    return ceph_location.replace('/', '\\')
```
Comment (gc-code) [inline WebApp/autoreduce_webapp/reduction_viewer/views.py]: Forward slashes should be used now that we are specifying a Linux path.

File: WebApp/autoreduce_webapp/templates/help.html [hunk 5]
```diff
@@ -118,7 +146,7 @@ <h4>Where does my reduced data go?</h4>
                 </p>
                 <p>
                     An example file path in CEPH would be:
-                    <code>\\data.analysis.stfc.ac.uk\rb\GEM\RBNumber\[RB NUMBER]\autoreduced\[RUN NUMBER]</code>.
+                    <code>\instrument\GEM\RBNumber\[RB NUMBER]\autoreduced\[RUN NUMBER]</code>.
```
Comment (gc-code) [inline WebApp/autoreduce_webapp/templates/help.html]: Forward slashes should be used now that we are specifying a Linux path.

**Comment by ElliotAOram** [comment]
@gc-code this should be ready to be reviewed again I think

**Comment by gc-code** [comment]
The link to reduced data still has backslashes.

**Comment by ElliotAOram** [comment]
interestingly this is how it is actually stored in the database. I'll go into the Queue Processor and change it

**Comment by ElliotAOram** [comment]
Coveralls failure due to remove a no longer required test. This should be ready to go now

**Pull request merged**